components/autosize_textarea.jsx

```diff
@@ -118,7 +125,7 @@ export default class AutosizeTextarea extends React.Component {
                         placeholder={placeholder}
                         rows='1'
                         {...otherProps}
-                        value={value || defaultValue || placeholder}
+                        value={hiddenValue}
```

Comment:
Any idea why `placeholder` is being passed into `value` in the first place?

Reply:
This textarea isn't the actual textarea, it's a second hidden one we use for sizing. Placeholder was getting passed in so that the sizing of an empty textarea would be correct

@hmhealey can probably expand more on why we have the hidden textarea, it's been a while and my memory is a bit fuzzy on the concrete reasoning for it

Reply:
That makes sense... but then why does the `placeholder` value passed into the hidden component result in a change to the "real" `textarea` `value`? Is this perhaps all related to the console log errors re: the `AutosizeTextArea`?

Reply:
It worked fine on every browser but IE11, my guess is it doesn't deal with disabled textareas very well and through some wonkiness caused it to affect the real value

Reply:
That warning is related but is just React complaining, we should probably fix it but I don't think it will have an affect on IE11

Reply:
The hidden textarea is for calculating the size of the content because the logic of setting the textarea to be the height of its content consistently was very messy and had edge cases before depending on different browsers. I think we passed in the `value={placeholder}` in because it was helping one of the browsers resize properly when you send a message and the textarea is cleared